#the header of the message
Collision[] collisions